fix_public.py

```
@@ -0,0 +1,29 @@
+files = [
+    "src/main/java/io/github/carlos_emr/carlos/managers/DemographicManager.java",
+    "src/main/java/io/github/carlos_emr/carlos/managers/LabManager.java",
+    "src/main/java/io/github/carlos_emr/carlos/PMmodule/service/ProgramManager.java",
+    "src/main/java/io/github/carlos_emr/carlos/casemgmt/service/CaseManagementManager.java",
+    "src/main/java/io/github/carlos_emr/carlos/managers/RxManager.java",
+    "src/main/java/io/github/carlos_emr/carlos/managers/PrescriptionManager.java",
+    "src/main/java/io/github/carlos_emr/carlos/managers/PreventionManager.java"
+]
+
+for fpath in files:
+    with open(fpath, "r") as f:
```

Ruff (0.15.12)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)